pkg/systemd/terminal.css

```
@@ -13,9 +13,35 @@ html, body {
     bottom: 0px;
     left: 0px;
     right: 0px;
-    background-color: black;
+}
+
+.black-theme {
+    background-color: #000000;
+}
+
+.dark-theme {
+    background-color: #002b36;
+}
+
+.light-theme {
+    background-color: #fdf6e3;
+}
+
+.white-theme {
+    background-color: #ffffff;
 }
 
 #terminal {
     height: 100%;
 }
+
+.terminal-group {
+    display: flex;
+    justify-content: space-between;
+}
+
+#theme-select {
+    margin-left: 5px;
+    margin-right: 5px;
+    width: auto;
```

```suggestion
    width: auto;
    min-width: 8rem;
```

Add a min-width of 8rem to make the theme selector not shift width when choosing another theme.